blockchain/types/transaction.go

```diff
@@ -294,19 +293,13 @@ func (tx *Transaction) EffectiveGasTip(baseFee *big.Int) *big.Int {
 	return tx.GasPrice()
 }
 
-// TODO-klaytn it use calculation algorithm of ethereum.
-// We have used it just like mock function. It returns governance unitprice
-// because GetGasTipCap() is same with GetGasFeeCap()
-// After hard fork, we need to consider new getEffectiveGasPrice API that operates differently
-// effectiveGasPrice = msg.EffectiveGasPrice(baseFee)
 func (tx *Transaction) EffectiveGasPrice(header *Header) *big.Int {
 	if header != nil && header.BaseFee != nil {
 		return header.BaseFee
 	}
 	if tx.Type() == TxTypeEthereumDynamicFee {
```

Comment:
```suggestion
	// Only enters if KIP-71 is not enabled. If KIP-71 is enabled, it will return BaseFee in the above if statement.
	if tx.Type() == TxTypeEthereumDynamicFee {
```

It would be good to add a comment for better understanding.